Fotogjengen/hilfling-frontend
# Feature/oscar/search page

**What has been done:**
-Added a seperate page for search
-Added search suggestions in frontend

**NOTE: This page does not display images, only the suggestions**

**How to test:**
-Start the backend
-Start the frontend and navigate to the "SØK" page
-Start to write "amber" and watch the suggestions appear (Make sure the DB is seeded)
-Good job!
